web/cypress/e2e/auto-tax-filing.spec.ts

```diff
@@ -18,35 +18,33 @@ describe("auto tax filing [feature] [all] [group4]", () => {
     cy.loginByCognitoApi();
     businessName = "Cool Business Name";
   });
-
-  it("automatically registers for Gov2Go and retrieves tax events if business name and tax id are provided", () => {
+  // There is an issue with circleCI where the taxId is entered wrongly as well. Need to investigate further.
```

Comment:
[sand] Updated the comment for clarity.

```suggestion
  // TODO: There is an issue in Cypress where the value of the taxId is being entered incorrectly by the automation. Need to investigate further. Temporarily skipping this test to avoid false failures.
```